torch/_inductor/decomposition.py

```diff
@@ -144,6 +144,23 @@ def convolution_backward(
     )
     return (grad_inp, grad_weight, grad_bias)
 
+@register_decomposition(aten.slice_scatter)
+def slice_scatter(self, src, dim=0, start=None, end=None, step=1):
+    if start == 0 and end >= 2**63 - 1 and step == 1:
+        return src
+    return NotImplemented
+
+@register_decomposition(aten.slice)
+def slice(self, dim=0, start=None, end=None, step=1):
+    if start == 0 and end >= 2**63 -1 and step == 1:
+        return self
+    return NotImplemented
+
+@register_decomposition(aten.mm)
+def mm(a, b):
+    if a.shape[0] == 1 or b.shape[1] == 1:
+        return (a.unsqueeze(2) * b.unsqueeze(0)).sum(dim=1)
```

Comment:
Same as #106593 (which already landed)?

Reply:
Will merge, but that one is for CPU only.

I'm actually gonna add this behind a flag I think - without coordinate descent autotuning this can lead to slowdowns.